### ISSUE TITLE
Support non `latin1` characters in file names

### ISSUE THREAD

**Issue by mks1948** [jaydenseric/graphql-upload#328]
After update from 14.0 to 15 i saw that is problem with non latin characters.
Package does not work correctly and after finish upload return some weird characters.
Please test with file name as :  你好.jpeg , معطل.jpeg or similar.

**Comment by jaydenseric** [comment]
If there is a problem with this, the [v15.0.0](https://github.com/jaydenseric/graphql-upload/releases/tag/v15.0.0) change most likely to be related would be the [`busboy`](https://npm.im/busboy) dependency being updated to v1.

> Package does not work correctly and after finish upload return some weird characters.

Can you please explain what happens more specifically? Where are the "weird characters".

**Comment by mks1948** [comment]
After upload file for example "你好.jpeg" i got a name as "ä½ å¥½.jpeg"

**Comment by jaydenseric** [comment]
What if you do:

```js
console.log(Buffer.from(strangeName, "latin1").toString("utf-8"));
```

Does it log the right way?

**Comment by mks1948** [comment]
Yup this log working correctly but i think that is not way to fix it in my code :)

**Comment by jaydenseric** [comment]
Yes, you shouldn't need to do that when this bug is fixed.

**Comment by jaydenseric** [comment]
This should be fixed in [v16.0.1](https://github.com/jaydenseric/graphql-upload/releases/tag/v16.0.1) 🚀

Be sure to remove any `Buffer.from(strangeName, "latin1").toString("utf-8")` workarounds before updating.
